code/style.css

```diff
@@ -1,4 +1,408 @@
 html {
-  background: #0025ff;
+  color: #000;
+  background-color: #f9f9f9;
+  font-family: 'Cutive Mono', monospace;
+}
+
+/* ------HEADER SECTION STYLING ----- */
+
+header {
+  overflow: hidden;
+  background: #F0F5F9;
+  font-family: 'Bebas Neue', cursive;
+  font-size: 1.5rem;
+  border-bottom: 8px solid yellow;
+}
+
+h1 {
+  font-size: 48px;
+  margin: 0;
+}
+
+.navigation {
+  background: #F0F5F9;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-right: 30px;
+}
+
+.left-links, .right-links {
+  display: none;
+  text-decoration: none;
+  color: #000;
+  font-size: 36px;
+  font-weight: 600;
+  padding: 20px;
+}
+
+.left-links:hover, .right-links:hover {
+  opacity: 0.6;
+}
+
+.logo {
+  width: 150px;
+  height: 150px;
+  padding: 10px;
+}
+
+
+/* ------BIG NEWS SECTION STYLING ----- */
+
+.big-news {
+  display: flex;
+  flex-direction: column;
+  margin: 30px;
+  padding: 20px;
+  background-color: #fff;
+}
+
+.big-news-text {
+  font-weight: lighter;
+  font-size: 16px;
+}
+
+hr {
+  height: 5px;
+  border: none;
+  background-color: rgb(59, 55, 55);
+  margin: 30px;
+}
+
+.big-news a {
+  text-decoration: none;
+  color: grey;
+}
+
+.big-news-image {
+  width: 100%;
+}
+
+.big-news:hover {
+  -webkit-transform: scale(1.005);
+  -moz-transform: scale(1.005);
+  transform: scale(1.005);
+  -webkit-box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
+  -moz-box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
+  box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
```

Comment:
Smart of you to use the -webkit- and -moz- to make sure it works in different browsers!

Reply:
After a bit googling I found out what they do :) Thank you!